pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
I meant the keyword `metrics` and not the API proposal itself.

I'm sorry, I don't get what `nodeId` refers to given that this API is at
the kubelet level?